python/tskit/drawing.py

```diff
@@ -554,6 +561,16 @@ def __init__(
                 self.add_class(self.mutation_label_attrs[m], "lab")
         self.draw()
 
+    @staticmethod
```

Comment:
These would be clearer as raw functions at the module level - ``self.identity()`` reads like "I want the identity of this object" and ``ClassName.identity()`` is similar.

Reply:
Good point. I initially defined them as lambdas, but black (or some other CI thing) didn't like that.